### ISSUE TITLE
Temporary files not cleaned up after crash and not easy to automate clean up on

### ISSUE THREAD

**Issue by phutchins** [Storj/core#167]
### Package Versions
```
/opt/storj/.nvm/versions/node/v5.10.1/lib
├── storj@1.2.0
└─┬ storj-bridge-client@0.4.4
  └── storj@0.6.16

```

Replace the values below using the output from `node --version`.

```
v5.10.1
```

### Expected Behavior

```
Temp files to clean themselves up or to be named such that it is easy to clean them up. Ideally, when running storj as a cli, it would check it's scratch space to see if there were any leftover files from a previous run and clean them up if they are old enough... Or in the case that the file upload or encryption operation does not finish, the cleanup operation still happens. This may not be possible if the process is killed however so having a consistent extension on them (i.e. 1dd92239a6a1.storj) might be helpful.
```

### Actual Behavior

```
1000's of tmp files get left in my /tmp directory over time and fill up my disk.
```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. Run storj upload-file or download-file repeatedly
2. If it crashes or does not finish, it may not clean up files
3. Check /tmp and you will see quite a few left over files

**Comment by bookchin** [comment]
I'll add a dir to `/tmp/storj`, that should make it easier to clean.

**Comment by bookchin** [comment]
Closed by #171
